scripts/Synapse/config/covid_identification_config.json

```diff
@@ -0,0 +1 @@
+{"covid_test_types": ["covid-19 test", "covid-19 antigen test", "covid-19 pcr test", "sars-cov-2 test", "sars-cov-2 antigen test", "sars-cov-2 genomic sequence test", "sars-cov-2 pcr test"], "covid_positive_results": ["positive", "detected", "detection of sars-cov-2 antigen in a clinical or post-mortem by any method", "detection of sars-cov-2 genomic sequence by any method", "detection of sars-cov-2 nucleic acid in a clinical or post-mortem specimen by any method", "Detection of sars-cov-2 organism or substance in a clinical specimen"], "covid_conditions": ["disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)", "covid", "coronavirus", "sars-cov-2", "severe acute respiratory syndrome coronavirus 2"]}
```

Comment:
I've definitely seen "POS" in addition to "positive" but I'm not sure if we want to edit on top of what LAC has given us.

Reply:
We haven't gotten these yet from LAC; they said end of day today (5/10). These values are currently just my best guess 😅 I will update once we get them from LAC.

Reply:
It will be better to have the list from their data set, but this isn't a bad start.